h2/src/main/org/h2/server/web/WebServer.java

```diff
@@ -54,6 +54,7 @@
         { "en", "English" },
         { "es", "Espa\u00f1ol" },
         { "fr", "Fran\u00e7ais" },
+        { "hi", "Hindi हिंदी" },
```

Comment:
Thank you!

Please, do not use national characters in `WebServer.java`. Replace them with unicode escape sequence like `\u0939\u093f\u0902\u0926\u0940`.

Reply:
Yes I will do.

Reply:
It's done now.